README.md

```diff
@@ -104,6 +124,13 @@ In the `"Serilog.WriteTo"` array, add an entry for `DatadogLogs`. An example is
   }
 }
 ```
+## Serilog.Sinks.Async
+As `Serilog.Sinks.Datadog.Logs` implements [PeriodicBatchingSink](https://github.com/serilog/serilog-sinks-periodicbatching), using [Serilog.Sinks.Async](https://github.com/serilog/serilog-sinks-async) is not recommended.
+
+From [Serilog.Sinks.Async documentation](https://github.com/serilog/serilog-sinks-async#serilogsinksasync---):
+> Note: many of the network-based sinks (CouchDB, Elasticsearch, MongoDB, Seq, Splunk...) already perform asynchronous batching natively and do not benefit from this wrapper.
+
+Note: There is a corner case where `Serilog.Sinks.Async` does not flush log correcly in version `v1.4.0`.
```

Comment:
```suggestion
Note: There is a corner case where `Serilog.Sinks.Async` does not flush log correctly in version `v1.4.0`.
```

and 2 questions:
1. Do you have a link to an issue/external link that describes this issue in more details? If so it would be relevant to reference it here.
2. Has this issue been fixed in later versions of the async plugin, or not yet? (again, would be interesting to mention that I think)

Reply:
After a deeper analysis, the issue was not in `Serilog.Sinks.Async` package but in `Serilog` package. It was fixed in the latest version by https://github.com/serilog/serilog/issues/1463.

Reply:
ok, thanks for looking into it! Since this was fixed recently, it might be worth mentioning this at the end of the README? (I don't see a reason not to, and it might help users avoid this bug in the future)